Sprint-3/implement/get-angle-type.js

```diff
@@ -25,3 +25,19 @@
 // Identify Reflex Angles:
 // When the angle is greater than 180 degrees and less than 360 degrees,
 // Then the function should return "Reflex angle"
+
+const getAngleType = (angleInDegree) => {
```

Comment:
Nice! I wonder, is there a way you could refactor this to make it a little simpler and clearer? Do you need all these elses?

Reply:
I have remove else part.